package.json

```diff
@@ -21,11 +21,12 @@
     "storybook:build": "build-storybook"
   },
   "dependencies": {
+    "@internetarchive/ia-icons": "1.1.2",
```

Comment:
_Question / Suggestion_ Are you using a bunch of the icons? If not, I might use the individual icons instead of `ia-icons`

Reply:
Yeah, I won't need it anymore. Replaced it with a recent commit.